app/frontend/components/domains/requirements-library/requirements-block-modal/index.tsx

```diff
@@ -29,7 +30,7 @@ export interface IRequirementBlockForm extends IRequirementBlockParams {
 }
 
 interface IRequirementsBlockProps {
-  requirementBlock?: IRequirementBlock
+  requirementBlock?: IRequirementBlock | IDenormalizedRequirementBlock
```

Comment:
this fixes some typescript errors regarding providing IDenormalizedBlock when IRequirementBlock is expected
This component was designed to accept either.